geothesaurus/index.php

```diff
@@ -137,41 +194,39 @@ function init() {
 		<div class="navpath">
 			<a href="../index.php">
 				<?= $LANG['NAV_HOME'] ?> </a> &gt;&gt;
-			<b> <?= $LANG['NAV_GEOTHES'] ?> </b>
+         <?php if($geoThesID): ?>
+			   <a href="index.php"><b> <?= $LANG['NAV_GEOTHES'] ?> </b></a>&gt;&gt;
+			   <b> <?= $geoUnit["geoTerm"]?> </b>
+         <?php else: ?>
+			   <b> <?= $LANG['NAV_GEOTHES'] ?> </b>
+         <?php endif ?>
 		</div>
 		<div id='innertext'>
+         <fieldset>
+            <legend><?=$LANG["SEARCH_GEOTHESAURUS"]?></legend>
+            <autocomplete-input 
```

Comment:
How very React-like. I had no idea something like this could be done in plain js. Very cool!